Assets/Group1/Scripts/GameManager.cs

```diff
@@ -0,0 +1,39 @@
+﻿using UnityEngine;
+using UnityEngine.Events;
+
+public class GameManager : MonoBehaviour
+{
+    public UnityEvent GameEnding;
```

Comment:
https://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods